release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -65,6 +65,16 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 * With this update, you can export your application in the ZIP file format to another project or cluster using the *Export application* option on the *+Add* page. 
 * With this update, you can create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
 
+[id="ocp-4-12-helm-page-improvements"]
+==== Helm page improvements
+
+In {product-title} 4.12, you can use the Helm page to:
```

Comment:
Is it possible that this "Helm page" has a slightly different name?
I'm not seeing any references to the "Helm page" in our current docs. The only related terms seem to be shown here: [helm-page-references.txt](https://github.com/openshift/openshift-docs/files/9786835/helm-page-references.txt).

Reply:
```
20:  . In the *Install Helm Chart* page:
  12:  . In the *Rollback Helm Release* page, select the *Revision* you want to rollback to and click *Rollback*.
  13:  . In the *Helm Releases* page, click on the chart to see the details and resources for that release.
  11:  . In the *Upgrade Helm Release* page, select the *Chart Version* you want to upgrade to, and then click
  11:  upgrade to, and then click *Upgrade* to create another Helm release. The *Helm Releases* page displays the two
  11:  then click *Upgrade* to create another Helm release. The *Helm Releases* page displays the two revisions.
```

Reply:
Please bold the name of the page, too.

Reply:
Thank you very much @rolfedh for letting me know about the Red Hat recommended way of writing release note entries. I shall rewrite all of them and send it for further review.

Reply:
No worries, Dhruv. Your work is good. We have many guidelines, and it takes quite a while to learn of them.

Reply:
Fixed.